submissions/fluidic-power/fluidic-power.md

```diff
@@ -67,12 +67,12 @@ caption: Table showing timing results of unmodified baseline benchmark compared
 
 ### Should you play it?
 
-Personally, I think [Fluidic Power](https://mods.factorio.com/mod/FluidicPower) creates a really fun challenge. It's maybe not a good fit you're planning on building a long duration megabase at the moment. However, if you're looking for an interesting [adventure playthrough](https://www.reddit.com/r/factorio/comments/qna0s7/comment/hjinkhd/?utm_source=share&utm_medium=web2x&context=3) with new challenges then this mod might be for you. I really enjoyed all the new obstacles to overcome. For example, the machines furthest away from your steam engines shuts down first, which is usually your coal patch which _feeds your boilers_. I've not experienced such a dangerous coal death spiral. Or if you defend with laser turrets you need to ensure there's enough transformers to supply power at the correct voltage.
+Personally, I think [Fluidic Power](https://mods.factorio.com/mod/FluidicPower) creates a really fun challenge. It's maybe not a good fit you're planning on building a long-lived megabase at the moment. However, if you're looking for an interesting [adventure playthrough](https://www.reddit.com/r/factorio/comments/qna0s7/comment/hjinkhd/?utm_source=share&utm_medium=web2x&context=3) with new challenges, then this mod might just be for you. I really enjoyed overcoming all the new obstacles it creates. For example, the machines furthest away from your steam engines shut down first, which is usually your coal patch which _feeds your boilers_. I've not experienced such a dangerous coal death spiral before. Or, if you defend with laser turrets, you need to ensure there's enough transformers to supply power at the correct voltage.
 
-However, my favourite part of this mod by far is knowing that there is actual electricity flowing through the power poles. You can even see the movement of intermittent brownouts when you're base is pulling a lot of power, like many laser turrets firing at once. It's similar to the feeling I get seeing a circuit I designed and knowing it's working because of the fluid-like motion of electrons through the copper and silicon. It builds on the feeling of seeing your complex networks of belts working together to build a rocket. It makes the factory feel more alive. You can basically see it's steady heartbeat, and it's really satisfying.
\ No newline at end of file
+However, my favourite part of this mod by far is knowing that there is actual electricity flowing through the power poles. You can even see the movement of intermittent brownouts when you're base is pulling a lot of power, like many laser turrets firing at once. It's similar to the feeling I get seeing a circuit I designed and knowing it's working because of the fluid-like motion of electrons through the copper and silicon. It builds on the feeling of seeing your complex networks of belts working together to build a rocket. It makes the factory feel more alive. You can basically see its steady heartbeat, and it's really satisfying.
```

Comment:
```suggestion
However, my favourite part of this mod by far is knowing that there is actual electricity flowing through the power poles. You can even see the movement of intermittent brownouts when your base is pulling a lot of power, like when many laser turrets fire at once. It's similar to the feeling I get when I look at a circuit I designed, while knowing it's working because of the fluid-like motion of electrons through the copper and silicon. It builds on the feeling of seeing your complex networks of belts working together to build a rocket. It makes the factory feel more alive. You can basically see its steady heartbeat, and it's really satisfying.
```
> You can even see the movement of intermittent brownouts

Maybe it's just me, but I don't really get what you mean with the word 'movement' here. How often brownouts occur? How machines start slowly shutting down? Isn't this something that also happens in vanilla brownouts as well, how is what you see any different from those? I didn't edit that part as I'm not sure what the original intention was.